README.md

```diff
@@ -369,11 +370,11 @@ This should do the trick. You can also check if there is any python at all via `
 Please remember `python2` is the one needed.
 
 
-#### known issues building on Windows
+#### Known issues building on Windows
 Right now there are few gotchas and the build will most likely fail. Please help with a PR if you know how to solve the issue, thank you!
 
 
-## Support
+## Features That Supported
```

Comment:
I think "Features that are supported" would be better

Reply:
Yes... exactly.... sorry for my english.. :smile: